static/js/containers/CreatePostPage.js

```diff
@@ -127,6 +135,7 @@ class CreatePostPage extends React.Component<CreatePostPageProps> {
           key:  actions.embedly.get.requestType
         }
       }
+      // $FlowFixMe
```

Comment:
why was this addition necessary? just curious

Reply:
With the addition of the catch block further below for posting, suddenly flow started complaining about:
```
Cannot call dispatch(...).then because property then is missing in object type [1].
```

¯\_(ツ)_/¯

Reply:
that's...bizarre

Reply:
Actually that was for the `dispatch.then` block below, which had been fine before.  For embedly the flow error was this:
```
Cannot call await with dispatch(...) bound to p because object type [1] is incompatible with Promise [
```